demo/csi/aws/efs/example.tf

```diff
@@ -0,0 +1,42 @@
+data "aws_kms_alias" "elasticfilesystem" {
+  name = "alias/aws/elasticfilesystem"
+}
+
+resource "aws_efs_file_system" "mytestefsvol" {
+  creation_token = "1eab7a71-2457-49b0-a56b-3519b21009a8"
```

Comment:
I know this is probably not a real token, but it's a good idea not to have hard-coded IDs anywhere in an example to avoid collisions if the person trying it out runs it again from fresh state. We can use the [`uuid`](https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/uuid) resource to generate this on the fly.

Reply:
The docs say terraform will provide one by itself if one is not given, so I'm just going to remove it.